### ISSUE TITLE
Decide to place a meeple over the river

### ISSUE THREAD

**Issue by zaovb** [farin/JCloisterZone#382]
From the problem of the rules that the river sticks to, I thought that a good (and small) option would be to allow the players to decide whether or not the river works as a border. This would allow those who play with the river as a non-border tile continue playing as they do at their tables.

I think the feature could be added in the "rules" section, so that the border can be enabled or disabled (keeping it as default).

**Comment by farin** [comment]
what do you mean "border". Are you playing river like it doesn't exists and field continues over it?

**Comment by zaovb** [comment]
Exactly. I, myself, like to play the river as a field border; but some people I have played with uses it just like it doesn't exist.

So, the feature I'm requesting is for those persons uses it like it doesn's exist.

Just enable or disable it.

**Comment by farin** [comment]
I see, it should be doable

**Comment by zaovb** [comment]
Thank you!

**Comment by stanolacko** [comment]
Hmm this is confusing but why not. I wonder hew it can work with some addons which uses River as feature. But usually they uses features on river not river at all.
